### PR TITLE
CFD-347 : Cannot edit my own discussion.

### PR DESCRIPTION
**Pull request by YuriGizra**
[CFD-347](https://issuetracker.amplexor.com/jira/browse/CFD-347)
#631

### CODE REVIEW

File: capacity4more/capacity4more.profile [hunk 1]
```diff
@@ -103,3 +108,32 @@ function capacity4more_setup_set_permissions(&$install_state) {
...
+    'photo',
+    'photoalbum',
+    'task',
+    'tasklist',
+    'wiki_page',
+  );
+
+  $permissions = array();
+  foreach ($content_types as $content_type) {
+    $permissions += array(
```
Comment (HelenaEksler) [inline capacity4more/capacity4more.profile]:
```php
$permissions = array_merge($permissions, array(
      "create $content_type content",
      "update own $content_type content",
      "delete own $content_type content",
    ));
```

**Comment by nader77** [comment]
@zero2one The tests are exceeding 50 mins and failing, How can we put them on the paid account ?

**Comment by nader77** [comment]
This is the OG permissions now, i think the Group admin should get the same permissions as well OR a permission to include all of them ? (Should be considered when granting permissions for QP)

og node group permissions capacity4more

**Comment by zero2one** [comment]
It's not very clear how to use the payed plan for our builds. I contacted Travis-CI about this. Waiting for feedback…

**Comment by nader77** [comment]
@zero2one PR is ready.

**Pull request merged**